flink-runtime/src/main/java/org/apache/flink/runtime/leaderelection/ZooKeeperLeaderElectionService.java

```diff
@@ -412,4 +413,11 @@ protected void handleStateChange(ConnectionState newState) {
 	public void unhandledError(String message, Throwable e) {
 		leaderContender.handleError(new FlinkException("Unhandled error in ZooKeeperLeaderElectionService: " + message, e));
 	}
+
+	@Override
+	public String toString() {
```

Comment:
Added because the instance logs itself:
`LOG.info("Starting ZooKeeperLeaderElectionService {}.", this);`